stps/sig-network/ip-request-stp.md

```diff
@@ -0,0 +1,218 @@
+| **Acceptance Criteria**                | [V]  | Ensured acceptance criteria are **defined clearly** (clear user stories; D/S requirements clearly defined in Jira).                                                                     |          |
+| **Non-Functional Requirements (NFRs)** | [ ]  | Confirmed coverage for NFRs, including Performance, Security, Usability, Downtime, Connectivity, Monitoring (alerts/metrics), Scalability, Portability (e.g., cloud support), and Docs. |          |
+
+- [KubeVirt Enhancements](https://github.com/kubevirt/enhancements/tree/main/veps)
+- [OCP Enhancements](https://github.com/openshift/enhancements/tree/master/enhancements)
+
+#### **2. Technology and Design Review**
+
+| Check                            | Done | Details/Notes                                                                                                                                           | Comments |
+|:---------------------------------|:-----|:--------------------------------------------------------------------------------------------------------------------------------------------------------|:---------|
+| **Developer Handoff/QE Kickoff** | [ ]  | A meeting where Dev/Arch walked QE through the design, architecture, and implementation details. **Critical for identifying untestable aspects early.** |          |
+| **Technology Challenges**        | [V]  | Identified potential testing challenges related to the underlying technology.                                                                           |          |
+| **Test Environment Needs**       | [V]  | Determined necessary **test environment setups and tools**.                                                                                             |          |
+| **API Extensions**               | [V]  | Reviewed new or modified APIs and their impact on testing.                                                                                              |          |
+| **Topology Considerations**      | [ ]  | Evaluated multi-cluster, network topology, and architectural impacts.                                                                                   |          |
+
+
+### **II. Software Test Plan (STP)**
+
+This STP serves as the **overall roadmap for testing**, detailing the scope, approach, resources, and schedule.
+
+#### **1. Scope of Testing**
+
+Briefly describe what will be tested. The scope must **cover functional and non-functional requirements**. Must ensure user stories are included and aligned to downstream user stories from Section I.
+
+**In Scope:**
+- Testing of known networking flows on VMs, where specific IP is requested for primary interface over
+Cluster/UserDefinedNetwork.\
+The uniqueness in this feature is how the IP is assigned - using the new `  network.kubevirt.io/addresses` annotation
```

Comment:
the annotation in the doc is `network.kubevirt.io/ip-addresses` - which one is the right one?